lightning/src/ln/chan_utils.rs

```diff
@@ -25,6 +25,25 @@ use secp256k1;
 pub(super) const HTLC_SUCCESS_TX_WEIGHT: u64 = 703;
 pub(super) const HTLC_TIMEOUT_TX_WEIGHT: u64 = 663;
 
+#[derive(PartialEq)]
+pub(crate) enum HTLCType {
+	AcceptedHTLC,
+	OfferedHTLC
+}
+
+impl HTLCType {
+	/// Check if a given tx witnessScript weight matchs one of a pre-signed HTLC
+	pub(crate) fn weight_to_htlctype(witness_script_len: usize) ->  Option<HTLCType> {
```

Comment:
Wait, why is this called "weight", its redeemScript length.

Reply:
The spec refers to weight (BOLT3), I always use it at so but yes you're right it's false. Keep witnessScript tho